src/components/Header/Header.tsx

```diff
@@ -62,13 +75,32 @@ export const Header = () => {
 
           <div className="header__buttons-block">
             <NavLink
+              onClick={() => {
+                setCurrentURL('favourites');
+              }}
               to="favourites"
               className={handleHeaderButtonIsActive}
             >
-              <img src="./logos/favourites.svg" alt="logoFavourite" />
+
+              {currentURL !== 'favourites'
+                ? (
+                  <img src="./logos/favourites.svg" alt="logoFavourite" />
+                ) : (
+                  <img
+                    src="./logos/favourites-selected.svg"
+                    alt="logoFavourite"
+                  />
+                )
+              }
```

Comment:
use spans, learn (button.scss - `&-favourite`, and change classNames when status changed)